packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -92,7 +92,7 @@ const defaultProps = {
   defaultValue: undefined,
   disabled: false,
   error: null,
-  errorIcon: Field.defaultProps.errorIcon,
+  errorIcon: <IconError/>,
```

Comment:
`defaultProps` is undefined since Field is wrapped in a HOC. Hence using `<IconError/>` directly instead of as a prop from Field.